### DISCUSSION TITLE
Is there a way to export a .qmd into a .ipynb?

### DISCUSSION THREAD

**Discussion by fred-spaceape** [posit-dev/positron#4826] [Q&A]
I'm not a huge fan of ipynb notebooks as you can't run individual lines of code, so I'm wondering if I can do my notebook-style analysis work in quarto and export as an ipynb notebooks (I want to be able to upload back into Databricks)

Is this possible in Positron?

**Comment by juliasilge** [comment]
This is sort of possible today, but there are some kinks for us still to work out. I'm not sure if enough is functional for you to move forward with the way you want to use this as of the current moment. 😬

- https://github.com/quarto-dev/quarto-cli/discussions/9463 outlines how you might convert from `.qmd` to `.ipynb` and some of the current limitations
- https://github.com/posit-dev/positron/issues/2866 outlines what we need to do so that you can use ark as a Jupyter kernel in Quarto

**Reply by fred-spaceape** [reply]
oh yeah that actually worked fine for me, thank you!
